tests/fixtures.py

```diff
@@ -21,6 +21,20 @@
     "MA190190000__2021_01_19_011931__C3.h5",
 )
 
+PATH_TO_GENERIC_0_4_2_FILE = os.path.join(
+    PATH_OF_CURRENT_FILE,
+    "h5",
+    "v0.4.2",
+    "MA190190000__2021_01_19_011931__C3__v0.4.2.h5",
+)
+
+TRIMMED_FILE_PATH = os.path.join(
```

Comment:
is there a benefit to having a trimmed file committed to the repository (as opposed to trimming it in a `session`-scoped fixture at the beginning of the test suite)?  in general, we want to try to commit the minimum number of files as possible to the git repository